data-sources/gigamon-appinsights/config.yml

```diff
@@ -0,0 +1,28 @@
+id: gigamon-appinsights
+displayName: Gigamon Application Insights
+description: |
+  Gigamon Application Intelligence gives NetOps, CloudOps, and security teams full visibility into applications running in datacenters and public clouds. Therefore, you can maintain consistently strong security across your environment plus get to the root causes of vulnerabilities and performance bottlenecks.
+icon: logo.png
+install:
+  primary:
+    nerdlet:
+      nerdletId: nr1-install-newrelic.quickstart-installation-plan
+      nerdletState:
+        quickstartId: d6b36ecf-bf99-475c-938f-370153db8e70
+      requiresAccount: true
+  fallback:
+    link:
+      url: https://www.gigamon.com/products/optimize-traffic/application-intelligence.html
```

Comment:
Since the quickstart is actually going to be displayed instead of this data source, let's update it to do the same thing as the install plan (we're swapping out install plans for data sources very soon)
```suggestion
install:
  primary:
    link:
      url: https://www.gigamon.com/products/optimize-traffic/application-intelligence.html
```